docs/articles/blog.md

```diff
@@ -0,0 +1,131 @@
+
+Web frameworks have realised long ago that there are a set of concerns that web applications must deal with such as connecting to databases, managing configuration, rendering static and dynamic content, and handling security concerns. Similar concerns need to be handled by machine learning projects but there are also specific concerns such as experimentation is a first class citizen, data and models need to be versioned and managed, model performance needs to be visualised, training infrastructure is required etc. Surround strives to provide a single place for every concern that arises when building a ML project. The concerns can face either the software engineer or research engineer and ideally there will only be a single solution for handling that problem. In situations where Surround does not support particular functionality it should be obvious how to add it.
+
+**Play Nice With Others**
+
+There are many good existing frameworks, libraries and APIs that research engineers can use to build a solution. Surround should strive to integrate nicely with as many of these 3rd party solutions as possible. This is reflected in the design of Surround where the core framework could be used to build: a solution based on cloud APIs, a custom Docker image for SageMaker or form part of a batch process running on an internal Kubernetes cluster. Research engieners also use different tools such as Jupyter notebooks which have a place in a Surround project (see below). By playing nice with others we hope the core Surround framework can continue to be used as the ML ecosystem continues to improve.
+
+**Familiarity**
+
+Many of the design guidelines and tools are heavily inspired by other frameworks such as Spring Boot for configuration management, Tensorflow serving for endpoint definition and the git command line tool for Surround's CLI. We made this a core philosophy to ensure both software engineers and research engineers can quickly get started with Surround. At times this was a challenge as Surround is a framework designed explicitly for two different types of people a traditional software engineer and a science focused research engineer.
+
+**Exploration to Production**
+
+From the moment data lands on our desk we need to be thinking about the final use case for the solutions we are developing. We built Surround to help us move from data exploration to a containerised proof-of-concept web application ready to deploy in the shortest time possible. To do this we needed to resolve the competing requirements of researchers and engineers. For example, researchers want to dive into the data leaving code quality for later where as engineers want well structured code from the start. We solved this problem by focusing the team on a "production first" mindset and through a convention (data exploration scripts had their own folder). At times small compromises needed to be made to ensure that everyone can use Surround for their use cases.
```

Comment:
maybe not resolve the competing requirements, but support the diverse needs of rsh eng + s/w eng? The language sounds adversarial.